### PR TITLE
Update message constraint for Transaction

### LINKED ISSUE #155: Update message contraints for Transaction amount field
Add a constraint that amount field should not contain commas

### PR DESCRIPTION
**Pull request by darkmoongreatsword**
Fixes #155

### CODE REVIEW

**Comment by kngys** [comment]
image

Maybe we should specify that there should be at least 1 digit before the decimal point (i.e. `.50` is invalid)?

**Comment by darkmoongreatsword** [comment]
> Maybe we should specify that there should be at least 1 digit before the decimal point (i.e. `.50` is invalid)?

That's a good idea. Added.

File: src/main/java/seedu/address/model/person/Transaction.java [hunk 1]
```diff
@@ -12,7 +12,12 @@
 public class Transaction {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Amount should have at most 2 decimal places.";
+            """
+            Amount should have at most 2 decimal places.
+            It should contain only digits(0-9) and one decimal point(.)
```
Comment (kngys) [inline src/main/java/seedu/address/model/person/Transaction.java:17]: Maybe can add a `if needed` to show the decimal point is optional

**Comment by kngys** [comment]
One minor thing should we say something to inform that the decimal place is optional? since integers like `10` are also accepted? In the user guide I wrote it like this 
* The amount should be a number of up to two decimal places containing only digits (`0 - 9`) and one decimal point (`.`) if needed 
* If the amount includes a decimal point, there should be at least one digit before the decimal point 
e.g. `0.5, 10, 100.5, 1000.55`

**Pull request merged**